modules/exploits/osx/local/vmware_fusion_lpe.rb

```diff
@@ -25,22 +25,22 @@ def initialize(info = {})
           with an effective UID of 0.
           @jeffball55 discovered an incomplete patch in 11.5.3 with a TOCTOU race.
           Successfully tested against 10.1.6, 11.5.1, 11.5.2, and 11.5.3.
-        ),
-        'License'        => MSF_LICENSE,
-        'Author'         =>
+        },
```

Comment:
Appears that RuboCop also picked up on the fact that `%q()` was being used rather than `%q{}`